web/server/models.py

```diff
@@ -305,4 +305,4 @@ def validate_schema(
         cls,
         v: list[tuple[str, str]] | list[tuple[exp.Column, exp.Column]],
     ) -> list[tuple[str, str]]:
-        return [(str(s), str(t)) for s, t in v]
+        return [(str(s).replace("s.", ""), str(t).replace("t.", "")) for s, t in v]
```

Comment:
My assumption that `str(s).replace("s.", "")` should work for both `list[tuple[str, str]]` and `list[tuple[exp.Column, exp.Column]]`

Reply:
The issue with replace is it doesn't only affect prefixes. I'm not sure if there are dialects of SQL where tables can have periods in their names, but this could be problematic if there were. There's `str.removeprefix` but that was introduced in Python 3.9.

Reply:
what are you trying to do? we shouldn’t use regex

Reply:
I need to return tuple with column names without "s" or "t"

Reply:
you can do that with sqlglot

Reply:
I'd remove the validator and pass in the column names (without the tables) as strings in the table_diff endpoint.

So `eq.left.name` and `eq.right.name` around here:
https://github.com/TobikoData/sqlmesh/blob/main/web/server/api/endpoints/table_diff.py#L59